### PR TITLE
Temorary Password reset to UIN

### PR DESCRIPTION
**Pull request by abgese**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -88,6 +88,11 @@ def create_reset_digest
     def send_password_reset_email
         UserMailer.password_reset(self).deliver_now
     end
+    
+    #Reset password to uin
+    def reset_password_to_uin
+        update_attribute(:password, self.uin)
```
Comment (harishk1908) [inline app/models/user.rb:94]: Any chance that a user exists without a UIN? Probably not, but best to be fully sure that UIN can't be empty lest some user has no way to reset their password.

**Review by harishk1908 — COMMENTED** [review]
Tested locally or was it tested by deploying to Heroku? Either is fine.

**Comment by abgese** [comment]
I tested locally

File: app/models/user.rb (hunk 1 above)
Comment (abgese) [reply app/models/user.rb:94]: In the 'user' model file: https://github.com/harishk1908/Project_Collection_and_Assignment/blob/master/app/models/user.rb

There is validation code in the beginning that ensures that the uin is a non-empty numerical value of max length 9. So I don't think there is any way for the uin to be empty.

Comment (harishk1908) [reply app/models/user.rb:94]: Thanks!

**Pull request merged**